boards/acd52832/Makefile.include

```diff
@@ -3,5 +3,6 @@ export CPU_MODEL = nrf52832xxaa
 
 # set default port depending on operating system
 PORT_LINUX ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
```

Comment:
no need to set this, its the default value defined in `nrf52/Makefile.include`, right?!